rpcs3/rpcs3qt/settings_dialog.ui

```diff
@@ -2275,13 +2275,25 @@
                </layout>
               </widget>
              </item>
+             <item>
+              <widget class="QGroupBox" name="gb_psnCountryBox">
+               <property name="title">
+                <string>Country</string>
+               </property>
+               <layout class="QVBoxLayout" name="gb_psnCountryBox_layout">
+                <item>
+                 <widget class="QComboBox" name="psnCountryBox"/>
+                </item>
+               </layout>
+              </widget>
+             </item>
```

Comment:
Please revert all the other changes.

Reply:
Do you mean just the portion you highlighted or everything other then this?

Reply:
Obviously I mean all the other random stuff

Reply:
Oh i see it now. My apologies. I didn't explicitly set any of them, as i configured it within QT Studio. Will revert those adjustments.